frame/elections-phragmen/src/benchmarking.rs

```diff
@@ -536,6 +536,9 @@ benchmarks! {
 	}
 }
 
+// can't use impl_benchmark_test_suite for this module because the tests depend on
```

Comment:
can't we pass in a custom exec name instead of assuming `execute_with`?

Reply:
Yes, we can; it took a while to figure out the appropriate syntax to use optional arguments in macros, but f1859ee1f should implement precisely that.